### ISSUE TITLE
Accessibility issue with  $govuk-secondary-text-colour	#6f777b 

### ISSUE THREAD

**Issue by TamsinProsser** [alphagov/govuk-design-system-backlog#202]
Hiya, 

My team has been using  $govuk-secondary-text-colour #6f777b as the hint text colour in a service we are working on. 

We did some accessibility testing using the [GDS empathy lab persona Ron](https://alphagov.github.io/accessibility-personas/ron/). When using the chrome extension recommended to simulate Ron's cataracts we were unable to read the text. 

We're concerned that the text colour may not be accessible for users who are sight-impaired.

**Comment by nickcolley** [comment]
Heya @TamsinProsser 

Thanks for raising this, we're capturing issues around secondary text colour over in a different repository.

So will close this out and quote you there:
https://github.com/alphagov/govuk-frontend/issues/1134
